src/api/internal.rs

```diff
@@ -1240,6 +1233,47 @@ impl<T: Pixel> ContextInner<T> {
         )
         .unwrap();
       }
+      let keyframe = fi.frame_type == FrameType::KEY;
+      let mut ref_log_isqrt_mean_scale = fi.ref_log_isqrt_mean_scale;
+      if output_framenos.len() > 1 {
+        let fi = &mut self
+          .frame_data
+          .get_mut(&output_framenos[1])
+          .unwrap()
+          .as_mut()
+          .unwrap()
+          .fi;
+        if keyframe && fi.frame_type != FrameType::KEY {
```

Comment:
Would this ever be true?

Reply:
Scoping makes this condition confusing, so I should make it clearer. The second clause refers to the next output frame. It is true unless there are consecutive keyframes.